src/server/db/migrations/20160217083000-simsupplier-migration.test.js

```diff
@@ -38,18 +38,12 @@ module.exports.up = function(db, config)
 
     // -----
 
-    // Load data.
-    const user2SupplierData = require(path + '/user2supplier.json');
```

Comment:
generally we should assume anything that is "live" can only be modified by additional migrations or in the worst case a rollback.
Means we are practising doing proper migrations already.
There are problems when not using query interface but sequelize models for injecting test data, so i guess we are down to direct query for data manipulation.
In any case I have seen examples of incomplete/bad rollback (down migration), so we should figure how to properly test and ensure it really does what it is supposed to do